content/blog/bare-metal-provisioning-on-hpe-greenlake-using-terraform.md

```diff
@@ -0,0 +1,359 @@
+The service can be accessed via GUI as well as via public APIs, enabling developers to use an Infrastructure-as-Code tool to build, change, and manage infrastructure in a consistent and repeatable way.
+
+## HPE GreenLake Terraform Provider
+
+The HPE GreenLake Terraform provider **hpegl** by HPE GreenLake provides Infrastructure-as-Code support for HPE GreenLake Cloud Services.
+Using the **hpegl** Terraform provider, you can automate the management of your infrastructure. You can provision OS on bare metal, spin virtual machines and bring up a Kubernetes cluster, starting right from bare metal all the way up in the stack to desired configurations and applications.
+
+In this blog post, I will walk you through the steps required to use the HPE GreenLake Terraform Provider to deploy and further manage bare metal compute instances.
+
+# Preparing for Infrastructure-as-Code implementation
+
+## Setting up API Client for access
+
+You need an API client to authenticate against the HPE GreenLake.
+
+Follow the below steps for API Client creation:
+
+1. From the HPE GreenLake platform, launch the HPE GreenLake Central console for the appropriate tenant. Under the settings icon on the tenant Dashboard page, select the **User Management** option. 
+
+![User Management](/img/greenlake_console_usermanagement.png "User Management")
+
+2. Under the API Clients tab, click on **Create API Client**.
+
+![Create API Client](/img/greenlake_console_createapiclient.png "Create API Client")
+
+3. Enter a **Name** (mandatory field) and **Description** (optional) for the API client, and click on **Create** button.
+
+![Create API Client](/img/greenlake_console_createapiclient_form.png "Create API Client")
+
+4. Ensure you make a note of the **Issuer**, **Client ID**, and **Client Secret** before clicking on the **Close** button. These details will be exported as environment variables in the next section.
+
+![API Client Created](/img/greenlake_conolse_apiclient_created.png "API Client Created")
+
+5. In the API Clients page, select the newly created client, and click on **Create Assignment** button.
+
+
+6. Assign the roles **BMAAS Access Viewer** and **BMAAS Access Project Contributor** on the **Space: Default.**
+
+![BMaaS Roles](/img/greenlake_console_createbmaasassignment.png "BMaaS Roles")
+
+The API Client is now ready to be used to run the Terraform resources.
+
+## Select the Compute Group ID
+
+Compute Group is a logical grouping of bare metal resources (Compute Instances, Networks, SSHKeys, etc.) that a team of Cloud Consumers can consume. You must specify the compute group ID to interact with the bare metal service resources. You can create a new compute group or select the existing one from the HPE GreenLake console. Make a note of the compute group ID because you need it to set a variable.
+
+***Note: Compute Group is AKA Project.***
+
+1. Navigate to HPE GreenLake for Private Cloud Services card -> Bare Metal -> Compute Groups.
```

Comment:
Because you are navigating screens that show it this way, I would not make any change here.